src/highrl/callbacks/robot_callback.py

```diff
@@ -301,12 +302,17 @@ def run_n_episodes(
         DataFrame: Logs for evaluation episodes.
     """
     env.opt.episode_statistics["scenario"] = "robot_env_test"
+    i = 0
```

Comment:
Rename variable to:
 - Have a more meaningful name
 - Conform to snake case rules